ruzanna.vanyan/repo/Lesson_3/exercise_13/exercise_13.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+cost = int(input("Enter the cost of your meal: "))
+tax = int(cost * 10 / 100)
+tip = int(cost * 18 / 100)
+total = int(cost + tax + tip)
```

Comment:
LW: If you convert to an integer you`re losing numbers after the dot(.) and it will cause loss of money.